Backout DINFOCACHE from di_init as it does not work for us (part of upstream
fe556c48dad96717b0ba71d489575c6b90b46448). Xorg fails to start with

  intel(0): intel_uxa_set_pixmap_bo: size of buffer object does not
  match constraints: size=14680064, must be greater than 14376960, but less than 0

and `scanpci -v` produces "di_init failed".

--- libpciaccess-0.14/src/solx_devfs.c	2017-10-23 17:48:29.000000000 +0000
+++ libpciaccess-0.14/src/solx_devfs.c.new	2019-03-09 19:04:03.462457526 +0000
@@ -543,7 +543,7 @@ pci_device_solx_devfs_probe( struct pci_
      * starting to find if it is MEM/MEM64/IO
      * using libdevinfo
      */
-    if ((rnode = di_init(nexus->dev_path, DINFOCACHE)) == DI_NODE_NIL) {
+    if ((rnode = di_init(nexus->dev_path, DINFOCPYALL)) == DI_NODE_NIL) {
 	err = errno;
 	(void) fprintf(stderr, "di_init failed: %s\n", strerror(errno));
     } else {
@@ -1075,7 +1075,7 @@ pci_system_solx_devfs_create( void )
 	return 0;
     }
 
-    if ((di_node = di_init("/", DINFOCACHE)) == DI_NODE_NIL) {
+    if ((di_node = di_init("/", DINFOCPYALL)) == DI_NODE_NIL) {
 	err = errno;
 	(void) fprintf(stderr, "di_init() failed: %s\n",
 		       strerror(errno));
